bulkwebhook/bulk_webhook/doctype/kafka_hook/kafka_hook.py

```diff
@@ -7,9 +7,11 @@
 from frappe import _
 from frappe.model.document import Document
 from frappe.utils.jinja import validate_template
-from frappe.utils.safe_exec import get_safe_globals
 from bulkwebhook.bulk_webhook.doctype.kafka_settings.kafka_utlis import send_kafka
 from bulkwebhook.bulk_webhook.doctype.bulk_webhook.bulk_webhook import log_request
+from frappe.integrations.doctype.webhook.webhook import get_context as get_webhook_context
```

Comment:
If I'm not mistaken imports are in a frappe-first sequence 😅

Reply:
Is this a convention for the project? The order doesn't really matter unless imports are circular, something is monkey patched, or another requirement, no? 

Anyway, eliminating this line altogether 😄

Reply:
Yeah, seems like a convention.

We do it for India Compliance as well. For custom apps, frappe imports can be thought of as "standard" imports, so I guess it's nice to have a differentiation.

Reply:
Just `isort` and forget :smile: